wowchemy/archetypes/slides/index.md

```diff
@@ -12,7 +12,42 @@ slides:
   theme: black
   # Choose a code highlighting style (if highlighting enabled in `params.toml`)
   #   Light style: github. Dark style: dracula (default).
-  highlight_style: dracula
+  #   Available highlight themes listed in: https://highlightjs.org/static/demo/
+  #   Use lower case names and replace space with hyphen '-'
+  highlight_style: monokai
+
+  mermaid_enabled: true # configure js or not
```

Comment:
There is an existing front matter option for this in Wowchemy, named `diagram`: https://wowchemy.com/docs/writing-markdown-latex/#diagrams . Let's try to be consistent.

Reply:
mermaid configuration can also be passed exactly the same way as reveal configuration.
Didn't make send to have
diagram: true to enable diagrams
mermaid: to configure options

its now set such that:
mermaid_enabled: true/false
mermaid:  to pass configuration
similar to menu_enabled and menu, chalkboard_enabled and chalkboard

pick your preferred type of consistency

Reply:
im going to change mermaid_option to diagram_options and explain in the comments that its mermaid options.
I'm not really sure if the diagram configuration is within scope of slides or not